app/serializers/event_serializer.rb

```diff
@@ -0,0 +1,6 @@
+class EventSerializer < ActiveModel::Serializer
```

Comment:
Yikes. I'm ok with the `camelCase`, but to optimize our payloads we're going to want the ability to send specific data from models. Seems like [jBuilder](https://github.com/rails/jbuilder) might be our best bet.